cmd/alias.go

```diff
@@ -0,0 +1,36 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+
+	"github.com/tfversion/tfversion/pkg/alias"
+)
+
+const (
+	aliasExample = "# Alias a Terraform version\n" +
```

Comment:
Maybe we can improve the text a bit to reflect it's like a "workspace" rather? 

I also quite like the word "workspace" in this context.

`$ tfversion workspace default`
`$ tfversion workspace hashicups`

Reply:
I think workspaces would be a different feature, like an isolated install of Terraform in your current directory?